### PR TITLE
Change abstain variable name to noConfidence when calculating vote results

### LINKED ISSUE #163: Change abstain variable name to noConfidence when calculating vote results
### In-Depth Feature Description:
This is specifically for a function called `getVotedPlayer` in `VoteHandler.js`.
`noConfidence` is a vote, whereas `abstain` means that a player has refused to vote at all. It would not appear as a entry in the `voteMap`. `noConfidence` would be a more appropriate name to represent entries in the votemap that have decided to go for no one to kill.

### Issue Approval:

- [x] Issue approved by: @SreenikethRaghavan

### PR DESCRIPTION
**Pull request by yzho976**
# Description
This PR fixes #[163]

This is specifically for a function called getVotedPlayer in VoteHandler.js.
noConfidence is a vote, whereas abstain means that a player has refused to vote at all. It would not appear as an entry in the voteMap. In this case, use noConfidence instead of abstain Vote.

# Demo
N/A

# Type of change:
(Delete the ones that are not relevant)
* Bugfix (Non-breaking change which fixes a bug)

# Checklist:
* Have you merged main into your branch?
* Have you tested your changes to ensure it works as expected and does not break existing functionality?
* If applicable, please ensure sufficient tests are added that is related to the changes.
* Please ensure this PR has a `label`, is linked to an `issue` and is related to a `project`
* If new documentation is required for this change, have you created a new documentation issue that describes the documentation needed?

### CODE REVIEW

**Comment by Milk-Yan** [comment]
Overlaps with #292

File: backend/.eslintrc.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
         "jasmine": true
     },
     "rules": {
+        "dot-notation": "off",
```
Comment (JChamberlain28) [inline backend/.eslintrc.json]: please remove this rule and reimplement with dot notation

**Review by JChamberlain28 — CHANGES_REQUESTED** [review]

**Review by khal745 — APPROVED** [review]
LGTM, but this work might get replaced with Yichengs work (#292) at a later stage. Thanks for this!

**Review by JChamberlain28 — APPROVED** [review]
Thanks for removing the extra eslint rule. Its important to keep rule changes for only when there is no other way to do things. Approved :)

**Pull request merged**